internal/databases/postgres/backup_sentinel_dto.go

```diff
@@ -77,28 +78,28 @@ type ExtendedMetadataDto struct {
 	UserData interface{} `json:"user_data,omitempty"`
 }
 
-func NewExtendedMetadataDto(isPermanent bool, dataDir string, startTime time.Time) (meta ExtendedMetadataDto, err error) {
+func NewExtendedMetadataDto(isPermanent bool, dataDir string, startTime time.Time,
+	sentinelDto BackupSentinelDto) (meta ExtendedMetadataDto) {
 	hostname, err := os.Hostname()
 	if err != nil {
-		return meta, err
+		tracelog.WarningLogger.Printf("Failed to fetch the hostname for metadata, leaving empty: %v", err)
```

Comment:
I replaced this error with a warning because we don't want entire backup to fail if we failed to obtain the hostname